overlord/snapstate/snapmgr.go

```diff
@@ -178,10 +179,72 @@ const (
 	NotBlocked
 )
 
+// RevisionSideInfo contains the side information for a snap and related components
+// installed in the system.
+type RevisionSideInfo struct {
```

Comment:
maybe this should be called RevisionSideState ?

Reply:
Changed